python/packages/aws-lambda-sdk/tests/fixtures/lambdas/sdk_sampled_out.py

```diff
@@ -7,7 +7,7 @@ def handler(event, context):
     counter += 1
     invocation_id = counter
 
-    from serverless_sdk import serverlessSdk as sdk
+    from sls_sdk import serverlessSdk as sdk
```

Comment:
I think here we can rely on `serverless_sdk` (I would stick to that name wherever we test public SDK, and do not access private modules in lib)